<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>530</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>530</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>700</width>
    <height>530</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>700</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Студенти</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>2903370.png</normaloff>2903370.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelInputGroupSize">
       <property name="text">
        <string>Введіть розмір групи</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineGroupSize">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Задати розмір</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>671</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Додати інформацію про студента</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>641</width>
           <height>381</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_Name"/>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>               Група</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>     Дата народження</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLineEdit" name="lineEdit_MoreInfo"/>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lineEdit_School"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>      Спеціальність</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Додаткова інформація</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QPushButton" name="addStudentButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Додати студента</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>                  ID</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QDateEdit" name="dateEdit"/>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="lineEdit_Speciality"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_Sname"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>           Прізвище</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>          По батькові</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="lineEdit_Group"/>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_Fname"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_ID"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>                Ім'я</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>   Навчальний заклад</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Змінити інформацію про студента</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>641</width>
           <height>381</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="4" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>           Прізвище</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QPushButton" name="EditButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Змінити інформацію</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Додаткова інформація</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>                  ID</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>     Дата народження</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLineEdit" name="lineEdit_Group_2"/>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>               Група</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>   Навчальний заклад</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>      Спеціальність</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="lineEdit_Speciality_2"/>
          </item>
          <item row="11" column="1">
           <widget class="QLineEdit" name="lineEdit_MoreInfo_2"/>
          </item>
          <item row="7" column="1">
           <widget class="QDateEdit" name="dateEdit_2"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>          По батькові</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="lineEdit_School_2"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>                Ім'я</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_Name_2"/>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_Sname_2"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_ID_2"/>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEdit_Fname_2"/>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>    Список студентів</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Інформація про студента</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>641</width>
           <height>381</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="9" column="0">
           <widget class="QLabel" name="label_46">
            <property name="text">
             <string>      Спеціальність</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_47">
            <property name="text">
             <string>          По батькові</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_41">
            <property name="text">
             <string>Додаткова інформація</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_42">
            <property name="text">
             <string>                  ID</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_44">
            <property name="text">
             <string>               Група</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_43">
            <property name="text">
             <string>     Дата народження</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_45">
            <property name="text">
             <string>   Навчальний заклад</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_40">
            <property name="text">
             <string>           Прізвище</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_49">
            <property name="text">
             <string>    Список студентів</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_48">
            <property name="text">
             <string>                Ім'я</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="comboBoxInfo"/>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLabel" name="label_get_ID">
            <property name="text">
             <string>Не вказано</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QLabel" name="label_get_Sname">
            <property name="text">
             <string>Не вказано</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QLabel" name="label_get_Name">
            <property name="text">
             <string>Не вказано</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QLabel" name="label_get_Fname">
            <property name="text">
             <string>Не вказано</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="QLabel" name="label_get_Date">
            <property name="text">
             <string>Не вказано</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="8" column="1" colspan="2">
           <widget class="QLabel" name="label_get_School">
            <property name="text">
             <string>Не вказано</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="9" column="1" colspan="2">
           <widget class="QLabel" name="label_get_Speciality">
            <property name="text">
             <string>Не вказано</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="10" column="1" colspan="2">
           <widget class="QLabel" name="label_get_Group">
            <property name="text">
             <string>Не вказано</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="11" column="1" colspan="2">
           <widget class="QLabel" name="label_get_MoreInfo">
            <property name="text">
             <string>Не вказано</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelError">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>20</y>
      <width>671</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>677</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QLabel" name="labelGroupSize">
       <property name="text">
        <string>-----------------------------------------------------------------[0/0]-----------------------------------------------------------------</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
